srcpkgs/nagios/files/nagios/run

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+chown -R _nagios:_nagios /var/nagios
```

Comment:
I don't like `chown -R` in run file.
On a big directory and slow filesystem, it will take a lot of time.

Reply:
This is an outdated commit - it has been replaced with `make_dirs` to specifically affect certain directories.